### DISCUSSION TITLE
Visual Basic Support in VS Code

### DISCUSSION THREAD

**Discussion by threatcon** [JosefPihrt/Roslynator#1300] [Q&A]
I'm new to the VS Code world, but I maintain a lot of legacy software projects at work. How hard would it be to include the Visual Basic tools? Even if it is a private fork, just curious on the amount of work that would be required. 

Also, thank you for your hard work on this project.

**Comment by josefpihrt** [comment]
- Analyzers would have to be written from the scratch (based on existing C# analyzers, where it makes sense)

- CLI generally works (or can work) for VB code. For example `analyze`/`fix` commands should work (if there are VB analyzers). `loc`/`lloc` commands also should work.

- Testing framework is available for VB but the packages have not been published yet.
